docs/migrating-from-0.9.md

```diff
@@ -0,0 +1,251 @@
+# Migrating from 0.9
```

Comment:
I think we need to mention the authPolicy change, and ToolDefinition being removed, the dotprompt  package change

Reply:
I think it would be helpful to discuss the new context concept in a bit more detail too. It seems like auth policy is a part of that, but it feels like an overall new concept that warrants some attention.

Reply:
Done.